src/pages/waypoint/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { ProductOption } from 'lib/learn-client/types'
+import TutorialView from 'views/tutorial-view'
+import {
+  getTutorialPagePaths,
+  getTutorialPageProps,
+  TutorialPageProps,
+  TutorialPagePaths,
+} from 'views/tutorial-view/server'
+
+export default function TutorialPage({
+  tutorial,
+}: TutorialPageProps): React.ReactElement {
+  return <TutorialView {...tutorial} />
+}
+
+export async function getStaticProps({
+  params,
+}): Promise<{ props: TutorialPageProps }> {
+  const props = await getTutorialPageProps(
+    ProductOption['waypoint'],
+    params.tutorialSlug
+  )
+  return props
+}
+
+export async function getStaticPaths(): Promise<{
+  paths: TutorialPagePaths[]
+  fallback: boolean
+}> {
+  const paths = await getTutorialPagePaths(ProductOption['waypoint'])
+  return { paths, fallback: false }
```

Comment:
Not doing ISR here yet. This function will only add 22 new waypoint tutorial routes to dev portal.